src/bentoml/_internal/service/inference_api.py

```diff
@@ -176,6 +176,9 @@ def _validate_route(route: str):
                 "Reserved API route: '{}' is reserved for infra endpoints".format(route)
             )
 
+    def __call__(self, *args: t.Any, **kwargs: t.Any) -> t.Any:
+        return self.func(*args, **kwargs)
```

Comment:
Idk if this is a good idea.

Reply:
How do we handle ctx in this case?

Reply:
user will need to pass in the context in that case 🤔

Reply:
hmm, that seems pretty hacky